### DISCUSSION TITLE
Feature: count for QUploader

### DISCUSSION THREAD

**Discussion by Hashen110** [quasarframework/quasar#12517] [Ideas / Proposals]
Is it possible to add a counter prop for QUploader to show no of selected files?

**Comment by MilosPaunovic** [comment]
You can easily achieve that using `header` slot:
[https://codepen.io/milospaunovic/pen/jOaZLoj?editors=1010](https://codepen.io/milospaunovic/pen/jOaZLoj?editors=1010)

Take note of this snippet:
```js
[{{scope.queuedFiles.length}} selected]
```

**Reply by Hashen110** [reply]
Thanks for the quick response :+1:

**Reply by Hashen110** [reply]
Can't we have a prop? Otherwise, we have to rewrite the whole `header` slot again
